CriticalMass/MessagesTableViewController.swift

```diff
@@ -36,12 +36,17 @@ class MessagesTableViewController<T: IBConstructableMessageTableViewCell>: UITab
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupTableView()
+    }
+
+    private func setupTableView() {
         // Setting the footerView hides seperators for empty cellls
         tableView.tableFooterView = UIView()
         tableView.register(cellType: T.self)
         // To use UITableViews dynamicHeight
         tableView.rowHeight = UITableView.automaticDimension
         tableView.estimatedRowHeight = 110.0
+        tableView.separatorColor = .gray300
```

Comment:
Why do we need a custom color here. Does that work with light/ dark mode?

Reply:
Works for both. 
I took a look the design and updated it

Reply:
have you tried overriding the userinterfacestyle depending on the current theme? That should fix the separator color without defining a custom one

Reply:
haven't tried that since it is only available in iOS 12 ⬆️ I think